neuralprophet/forecaster.py

```diff
@@ -625,13 +626,17 @@ def add_country_holidays(self, country_name, lower_window=0, upper_window=0, reg
         self.config_country_holidays.init_holidays()
         return self
 
-    def add_seasonality(self, name, period, fourier_order):
+    def add_seasonality(self, name, period, fourier_order, condition_name=None):
         """Add a seasonal component with specified period, number of Fourier components, and regularization.
 
         Increasing the number of Fourier components allows the seasonality to change more quickly
         (at risk of overfitting).
         Note: regularization and mode (additive/multiplicative) are set in the main init.
 
+        If condition_name is provided, the dataframe passed to `fit` and
```

Comment:
let's also support  binary values and values from 0 to 1 (that sum up to 1 for each timestamp).